File: 1. Model/Debito.py


class Debito:

    def __init__(self, valorDebito, dataDebito, descDebito, produto):
        self.valorDebito = valorDebito
        self.dataDebito  = dataDebito
        self.descDebito  = descDebito
        self.produto     = produto





    #  - - - - - - - - - - GETS & SETS - - - - - - - - - -
    ##GETS
    def getValorDebito(self):
        return self.valorDebito

    def getDataDebito(self):
        return self.dataDebito

    def getDescDebito(self):
        return self.descDebito

    def getProduto(self):
        return self.produto


    ##SETS
    def setValorDebito(self, vd):
        self.valorDebito = vd

    def setDataDebito(self, dtd):
        self.dataDebito = dtd

    def setDescDebito(self, dsd):
        self.descDebito = dsd

    def setProduto(self, p):
        self.produto = p

    # - - - - - - - - - - //// /  //// - - - - - - - - - -

File: model/Debito.java
package model;

//import java.util.Date;



public class Debito {

	private String dataDebito;

	private String produto;

	private String descDebito;

	private double valorDebito;



	public Debito(String d, String p, String s, double v) {
		this.dataDebito  = d;
		this.produto 	 = p;
		this.descDebito  = s;
		this.valorDebito = v;
	}



	// Gets
	public String getDataDebito() {
		return this.dataDebito;
	}

	public String getProduto() {
		return this.produto;
	}

	public String getDescDebito() {
		return this.descDebito;
	}

	public double getValorDebito() {
		return this.valorDebito;
	}



	// Sets

	public void setDataDebito(String d) {
		this.dataDebito = d;
	}

	public void setProduto(String p) {
		this.produto = p;
	}

	 public void setDescDebito(String s) {
		 this.descDebito = s;
	 }

	 public void setValorDebito(double v) {
		 this.valorDebito = v;
	 }
}
